main/src/ca/uwaterloo/flix/language/phase/Simplifier.scala

```diff
@@ -53,7 +53,7 @@ object Simplifier {
   private def visitEnum(decl: LoweredAst.Enum)(implicit flix: Flix): SimplifiedAst.Enum = decl match {
     case LoweredAst.Enum(_, ann, mod, sym, _, _, cases0, enumType, loc) =>
       val cases = cases0.map {
-        case (tag, LoweredAst.Case(caseSym, tagType, _, tagLoc)) => tag -> SimplifiedAst.Case(caseSym, visitType(tagType), tagLoc)
+        case (tag, LoweredAst.Case(caseSym, _, _, tagLoc)) => tag -> SimplifiedAst.Case(caseSym, null /* unused */, tagLoc)
```

Comment:
Say what? Null in my compiler! What is this.

Reply:
I figured a hard signal like `null` would be better than `Unit` :P

Reply:
We never use null. Why is there no real value here?

Reply:
For Option.Some the type is `t` - but MonoDefs/Types promises to ground the types